modules/archetypes/lib/policy_assignments/policy_assignment_es_deny_public_endpoints.tmpl.json

```diff
@@ -1,22 +1,23 @@
 {
-  "name": "Deny-Public-Endpoints",
   "type": "Microsoft.Authorization/policyAssignments",
-  "apiVersion": "2019-09-01",
+  "apiVersion": "2022-06-01",
+  "name": "Deny-Public-Endpoints",
+  "location": "${default_location}",
+  "dependsOn": [],
   "properties": {
-    "description": "This policy initiative is a group of policies that prevents creation of Azure PaaS services with exposed public endpoints.",
+    "description": "This policy initiative is a group of policies that prevents creation of Azure PaaS services with exposed public endpoints",
     "displayName": "Public network access should be disabled for PaaS services",
-    "notScopes": [],
-    "parameters": {},
     "policyDefinitionId": "${root_scope_resource_id}/providers/Microsoft.Authorization/policySetDefinitions/Deny-PublicPaaSEndpoints",
+    "enforcementMode": null,
     "nonComplianceMessages": [
       {
         "message": "Public network access {enforcementMode} be disabled for PaaS services."
       }
     ],
+    "parameters": {},
     "scope": "${current_scope_resource_id}",
-    "enforcementMode": "Default"
```

Comment:
Why is this set to Default?

Reply:
@matt-FFFFFF and @jtracey93 I am not sure if this was by design. It does look like it would override the setting in the code. Any thoughts?